LXReorderableCollectionViewFlowLayout/LXReorderableCollectionViewFlowLayout.m

```diff
@@ -353,8 +360,13 @@ - (void)handleLongPressGesture:(UILongPressGestureRecognizer *)gestureRecognizer
                  delay:0.0
                  options:UIViewAnimationOptionBeginFromCurrentState
                  animations:^{
+                     
                      __strong typeof(self) strongSelf = weakSelf;
-                     if (strongSelf) {
+                     if ( strongSelf.reorderCompletionAnimationBlock != nil ) {
+                         strongSelf.reorderCompletionAnimationBlock( strongSelf.currentView );
+                         strongSelf.currentView.center = layoutAttributes.center;
+                     }
+                     else {
```

Comment:
You might want to add an `else if ( strongSelf )` to maintain the original `if (strongSelf)` guard